internal/runners/config/config.go

```diff
@@ -18,13 +23,103 @@ type primeable interface {
 	primer.Analyticer
 }
 
-func NewConfig(prime primeable) (*Config, error) {
-	return &Config{
+func NewList(prime primeable) (*List, error) {
```

Comment:
Please store prime, eg. https://github.com/ActiveState/cli/blob/d43bc4921ea8fccf7f1848f4ba2e22004121e145/internal/runners/upgrade/upgrade.go#L46-L46

After the runtime refactor this is the new pattern I'd like to move us to. We still need the primeable interface but at least we are removing a bit of symbol soup this way.